db-migrations/src/main/resources/db/migration/V35__arrangor_veileder.sql

```diff
@@ -0,0 +1,10 @@
+CREATE TABLE arrangor_veileder (
```

Comment:
Trenger vi en kobling til hvilket tiltak/gjennomføring veileder-relasjonen gjelder? Man kan kanskje se for seg at en deltaker vil delta på flere tiltak hos samme arrangør, men ikke nødvendigvis ha samme veileder. Eller deltar man kun på et tiltak av gangen?

Reply:
En deltaker er tilknyttet kun ett tiltak/gjennomføring, en bruker kan være flere deltakere.